Automattic/wc-calypso-bridge
# Remove useSlot monkey patch

### Changes proposed in this Pull Request:



Closes #1081.

This PR removes the [useSlot monkey patch](https://github.com/Automattic/wc-calypso-bridge/commit/5c4b0550fd27360f32ad3ff111a5abd39d957e9e).

> Bridge has a [useSlot patch](https://github.com/Automattic/wc-calypso-bridge/blob/master/src/use-slot-patch.js) that fixes a breaking change introduced in Gutenberg. Since atomic now targets WooCommerce 7.6, the patch is no longer needed.






### How to test the changes in this Pull Request:




1. Go to `WooCommerce > Home`
2. Confirm the free trial banner exists



### Other information:




### FOR PR REVIEWER ONLY:
